systemd/pivot.service.in

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Pivot Tool
+ConditionPathExists=/etc/pivot/image-pullspec
+After=ignition-firstboot-complete.service
+Before=kubelet.service
+
+[Service]
+Type=simple
+ExecStart=@@PIVOT_BINARY_PATH@@
```

Comment:
Are you intentionally not doing `RemainAfterExit=yes` so it can be invoked multiple times?

Reply:
Not intentionally. Are you arguing for switching it to `Type=oneshot` + `RemainAfterExit=yes` in case other services start ordering themselves wrt to pivot.service? Feels like something we can defer until a valid use case comes up?

Reply:
Yeah, don't need to do this now.